src/component/plotly-dash-preview/parse.js

```diff
@@ -25,24 +27,29 @@ function parse (body, opts, sendToRenderer) {
   }
 
   result.pdfOptions = body.pdf_options || {}
-  if (!body.loading_selector && !body.timeout) {
-    return errorOut(400, 'either loading_selector or timeout must be specified')
+  if (!isNonEmptyString(body.selector) && !isPositiveNumeric(body.timeout)) {
+    return errorOut(400, 'either selector or timeout must be specified')
```

Comment:
This here may be overly stringent. Perhaps we could default to a 3000ms timeout if `timeout` and `selector` are omitted from the POST request. @chriddyp what do you think?